plugins/base/src/main/resources/dokka/styles/style.css

```diff
@@ -783,92 +1028,78 @@ small {
     margin: 0;
 }
 
-.platform-tag {
-    display: flex;
-    flex-direction: row;
-    padding: 4px 8px;
-    height: 24px;
-    border-radius: 100px;
-    box-sizing: border-box;
-    border: 1px solid transparent;
-    margin: 2px;
-    font-family: Inter, Arial, sans-serif;
-    font-size: 12px;
-    font-weight: 400;
-    font-style: normal;
-    font-stretch: normal;
-    line-height: normal;
-    letter-spacing: normal;
-    text-align: center;
-    outline: none;
-    color: #fff
+[data-filterable-current=''] {
+    display: none !important;
 }
 
-.platform-tags {
+.platform-tags, .filter-section {
     display: flex;
     flex-wrap: wrap;
-    padding-bottom: 8px;
+    margin-bottom: -8px;
+    margin-left: -4px;
 }
 
-.platform-tags > .platform-tag {
-    align-self: center;
-}
+.platform-tag {
+    border: 0 none;
+    border-radius: 4px;
+    padding: 2px 16px;
+    margin-right: 4px;
+    margin-bottom: 8px;
 
-.platform-tags > .platform-tag:first-of-type {
-    margin-left: auto;
+    font-family: "JetBrains Sans", Inter, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Droid Sans", "Helvetica Neue", Arial, sans-serif;
```

Comment:
Again, minor one.
I'm not sure it is worth specifying here JebBrains Sans and Inter if we don't add these fonts to the bundle. Considering that the system font is used for the content, I suggest removing these two from the begging.